addons/website_event/controllers/main.py

```diff
@@ -356,21 +442,23 @@ def _create_attendees_from_registration_post(self, event, registration_data):
 
             registrations_to_create.append(registration_values)
 
-        return request.env['event.registration'].sudo().create(registrations_to_create)
+        registrations_to_update.flush_recordset()  # CACL: flush to avoid access rights error, but don't know why...
```

Comment:
I've tried a lot to remove this but I have no more idea to solve this. From the traceback , it seems to happen just before the redirect on `registration/success` when we try to access the registrations from env not anymore in sudo.